modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/openapi/Oauth2OpenApiConfig.java

```diff
@@ -26,7 +26,7 @@
 /**
  * Configuration springdoc security oauth2.
  */
-@Profile("zitadel")
+@Profile({"zitadel", "oauth"})
```

Comment:
same here. we should be able to start this class without "oauth"
We need Zitadel only to set variables.